### DISCUSSION TITLE
How to specify resources for wait container?

### DISCUSSION THREAD

**Discussion by scape7yu** [argoproj/argo-workflows#6372] [Q&A] [answered]
I want the container wait execute more quickly, so I need to enlarge cpu of container wait, can you tell me where to define the resources like cpu and memory?

**Comment by scape7yu** [comment]
I see that the resources of container wait is from Kubernetes's LimitRange which defines Default Request and Default Limit.

**Comment by tczhao** [comment, marked as answer]
Hi @scape7yu you can use podspecpatch to udpate resource request for wait container
eg.
https://github.com/argoproj/argo-workflows/blob/master/examples/pod-spec-patch.yaml

**Reply by scape7yu** [reply]
oh, it works, my container wait runs very fast now, thank you so much!
